src/FE/src/app/pages/users-page/users-page.component.html

```diff
@@ -0,0 +1,104 @@
+<div class="container">
+  <div class="infoContainer">
+    <div class="userContainer">
+      <div class="userTitleContainer">
+        <p>{{ 'USERS' | translate }}</p>
+        <mat-form-field class="searchInput" appearance="outline" (keyup)="applySearch($event)">
+          <mat-label>{{ 'SEARCH_PROMPT' | translate }}</mat-label>
+          <input matInput inputmode="search" type="search">
+          <mat-icon matPrefix>search</mat-icon>
+        </mat-form-field>
+      </div>
+
+      <div class="tableContainer">
+        <table class="userTable" mat-table [dataSource]="datasource">
+          <!-- name -->
+          <ng-container matColumnDef="username">
+            <th mat-header-cell *matHeaderCellDef>
+              {{ 'USERNAME' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+            <span>
+              {{ element['username'] }}
+            </span>
+            </td>
+          </ng-container>
+
+          <!-- Change password button column -->
+          <ng-container matColumnDef="changePassword">
+            <th mat-header-cell *matHeaderCellDef>
+              {{ 'CHANGE_PASSWORD' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let row">
+              <button mat-icon-button (click)="showChangePassword(row)">
+                <mat-icon>lock_reset</mat-icon>
+              </button>
+            </td>
+          </ng-container>
+
+          <!-- Delete button column -->
+          <ng-container matColumnDef="delete">
+            <th mat-header-cell *matHeaderCellDef>
+              {{ 'DELETE' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let row">
+              <button mat-icon-button (click)="onDelete(row)">
+                <mat-icon>delete</mat-icon>
+              </button>
+            </td>
+          </ng-container>
+
+          <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+          <tr mat-row *matRowDef="let row; columns: displayedColumns" (click)="selectRow(row)"></tr>
+        </table>
+      </div>
+    </div>
+
+    <div *ngIf="userForm" class="newUserContainer" [formGroup]="createUserForm">
+      <p>{{ 'NEW_USERS' | translate }}</p>
+      <mat-form-field>
+        <mat-label>{{ 'USERNAME' | translate }}</mat-label>
+        <input matInput formControlName="username">
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>{{ 'PASSWORD' | translate }}</mat-label>
+        <input matInput formControlName="password" type="password">
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>{{ 'REPEAT_PASSWORD' | translate }}</mat-label>
+        <input matInput formControlName="repeatPassword" type="password">
+      </mat-form-field>
+
+      <button mat-raised-button (click)="onSave()">
```

Comment:
tilføj [disable]="<FORMGROUP>.invalid". på den måde skal felterne være udfyldt korrekt for at kunne trykke reset.

ps. erstat med navnet på den formgroup, der bruges her